src/api/status.js

```diff
@@ -106,16 +106,11 @@ export async function status({
         // If the oid in the index === working dir oid but stats differed update cache
         if (I && indexEntry.oid === workdirOid) {
           // and as long as our fs.stats aren't bad.
-          // size of -1 happens over a BrowserFS HTTP Backend that doesn't serve Content-Length headers
-          // (like the Karma webserver) because BrowserFS HTTP Backend uses HTTP HEAD requests to do fs.stat
-          if (stats.size !== -1) {
```

Comment:
You should not remove this code, the documentation may use ZenFS, but users may still use BrowserFS even that's deperecated. We don't want breaking changes.

Reply:
https://github.com/isomorphic-git/isomorphic-git/pull/1996/commits/7ccd61703d9a2a85c04d32f5320752e11fc215e7